http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -162,6 +162,12 @@ public void testTokenSignatureAlgorithm() throws Exception {
                 true, HttpStatus.SC_MOVED_TEMPORARILY, getClientUrl(), CLIENT_PAGE_TEXT);
     }
 
+    @Test
+    public void testScope() throws Exception {
```

Comment:
Would be good to add tests for the following cases:

* invalid scope value
* scope value with one scope specified
* scope value with multiple scopes specified

Reply:
We should also try to check if the Keycloak OpenID provider supports any scope values that we could try to configure in the test.

Reply:
Implemented reviews upto this comment in commit 3ee9bb6b88d6a4748edbf012b9ff9648e4f437d6